driver-sync/src/main/com/mongodb/client/internal/MongoIterableImpl.java

```diff
@@ -116,26 +116,18 @@ public TResult first() {
         return new MappingIterable<TResult, U>(this, mapper);
     }
 
-    @SuppressWarnings("overloads")
-    private void forEach(final Block<? super TResult> block) {
```

Comment:
No longer needed since we can use the new method that overrides the one in Iterable